src/containers/booksForm.js

```diff
@@ -1,11 +1,80 @@
-/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable class-methods-use-this */
 /* eslint-disable no-plusplus */
+
 import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { createBook } from '../actions/index';
+
+class BooksForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      category: '',
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSelect = this.handleSelect.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    event.preventDefault();
+    this.setState({
+      title: event.target.value,
+    });
+  }
+
+  handleSelect(event) {
+    event.preventDefault();
+    this.setState({
+      category: event.target.value,
+    });
+  }
+
+  validate(title, category) {
+    const formError = document.getElementById('formError');
+    if (title === '') {
+      formError.innerHTML = "Title can't be empty";
+      setTimeout(() => {
+        formError.innerHTML = '';
+      }, 3000);
```

Comment:
**`Optional`** Basing the error message on a timeout is not a good practice, you could use `onBlur` which is more like `offFocus`.

Reply:
DONE
used onFocus of fields